jekyll/_cci2/dynamic-config.md

```diff
@@ -6,48 +6,48 @@ version:
 
 ## Getting started with dynamic config in CircleCI
 {: #getting-started-with-dynamic-config-in-circleci }
 
 To get started with Dynamic Config in CircleCI:
 
-- Select the project you are interested in, in the **Projects** dashboard from the CircleCI application.
-- Click the **Project Settings** button in the upper-right corner.
-- On the left-hand panel, select **Advanced**.
-- Towards the bottom, toggle the switch for **Enable dynamic config using setup workflows** to the "on" position, as shown below:
+1. In the **Projects** dashboard from the CircleCI web application, select your project.
+2. Click the **Project Settings** button in the upper-right corner.
+3. On the left-hand panel, select **Advanced**.
+4. Scroll to the **Enable dynamic config using setup workflows** setting, and toggle it to the "on" position, as shown below:
+  <br>
+  ![Enable dynamic config in the UI]({{ site.baseurl }}/assets/img/docs/dynamic-config-enable.png)
 
-![Enable dynamic config in the UI]({{ site.baseurl }}/assets/img/docs/dynamic-config-enable.png)
+5. While the steps above will make the feature available, your static `config.yml` will continue to work as normal. This feature will **not** be used until you add the `setup` key with a value of `true` to that `config.yml`.
+  Adding the key `setup: true` to the top level of your parent configuration file (in the `.circleci/` directory) will designate that `config.yml` as a setup configuration.
+6. At the end of the `setup` workflow, a `continue` job from the [`continuation` orb](https://circleci.com/developer/orbs/orb/circleci/continuation) must be called. **NOTE:** This does _not_ apply if you want to conditionally execute workflows or steps based on **updates to specified files**, as described in the [Configuration Cookbook]({{ site.baseurl }}/2.0/configuration-cookbook/?section=examples-and-guides#execute-specific-workflows-or-steps-based-on-which-files-are-modified) example.
 
-Now, your project has the ability to dynamically generate and update configuration.
+## How dynamic config works
+{: #how-dynamic-config-works }
 
-Note: While the steps above will make the feature available, your static `config.yml` will continue to work as normal. This feature will not be used until you add the key `setup` with a value of `true` to that `config.yml`.
+CircleCI's dynamic configuration feature uses setup workflows. A setup workflow can contain jobs that compute pipeline parameters, which can then be passed into an additional configuration that potentially exists in other directories. A setup workflow can also be used to generate follow-up pipelines via pre-existing scripts.
```

Comment:
I wouldn't talk about follow-up pipelines, they're a different and confusing concept in this context.

Reply:
On the other hand it might be worth mentioning that not only pipeline parameters can be generated, the entire config can be generated within the setup workflow.